_posts/2017/2017-02-23-scrum-interview.md

```diff
@@ -0,0 +1,140 @@
+---
+layout: post
+title: "Scrum Master insights at ePages"
+date: "2017-02-09 07:00:00"
+categories: working-at-epages
+authors: ["Rose", "Ann-Katrin"]
+---
```

Comment:
Please agree on a header image with Ann-Katrin.
If you don't want to go with a picture of her, you can maybe use one of those
https://www.pexels.com/photo/ideas-whiteboard-person-working-7369/
https://www.pexels.com/photo/woman-in-gray-shirt-seating-in-between-woman-in-blue-black-and-white-plaid-long-sleeve-shirt-and-man-in-blue-white-and-black-plaid-long-sleeve-shirt-having-conversation-57825/
https://www.pexels.com/photo/startup-planning-notes-mac-book-7357/